pycobertura/reporters.py

```diff
@@ -54,6 +54,13 @@ def get_report_lines(self):
 
         return lines
 
+    def per_file_stats(self, lines):
```

Comment:
The reading of this function is a little cryptic. Any chance we can replace `k` and `v` with more descriptive names? Maybe `k` could be `header_name` or something like that?

Also, would it be helpful to have a docstring that illustrates what will be returned by this function so readers understand what to expect?

Reply:
I like to use an ellipsis when it makes sense to avoid being too verbose, such as:

```python
def per_file_stats(self, lines):
    """
    Returns dict with keys `files` and `total` that contain coverage
    statistics per file and overall, respectively.

    {
        "files": [
            {
                "Filename": "dummy/__init__.py",
                "Stmts": 0,
                "Miss": 0,
                "Cover": "0.00%",
                "Missing": "9"
            },
            ...
        ],
        "total": {
            "Filename": "TOTAL",
            "Stmts": "...",
            "Miss": "...",
            "Cover": "..."
        }
    }
    """
    # <... code ...>
```

What do you think?

Reply:
Also, I'd like to rename `lines` to something less ambiguous. `lines` here is the output of `get_report_lines` which is quite different than `line` used elsewhere in the code base.

Reply:
done in recent commit 1c6d2da62c48fb443daf4d3c9350787fc23f1a5a.